RoaringBitmap/src/main/java/org/roaringbitmap/BitmapContainer.java

```diff
@@ -869,7 +869,8 @@ public Container ixor(final ArrayContainer value2) {
   public Container ixor(BitmapContainer b2) {
     // do this first because we have to compute the xor no matter what, and this loop gets
     // vectorized and is faster than computing the cardinality or filling the array
-    for (int k = 0; k < this.bitmap.length & k < b2.bitmap.length; ++k) {
+    int overlap = Math.min(this.bitmap.length, b2.bitmap.length);
+    for (int k = 0; k < overlap; ++k) {
```

Comment:
bitmaps always have the same length (1024) so there is no way this could speed the loop up.

Reply:
as I wrote above `I suspect that some optimizations might kick in or speculation is simpler for cpu.`, this statement relates to having simpler condition in a loop. Could you run benchmarks on yours machine to either confirm or deny these scores that I have posted ?